sinks/api/supported_labels.go

```diff
@@ -22,6 +22,7 @@ const (
 	LabelLabels        = "labels"
 	LabelHostname      = "hostname"
 	LabelResourceID    = "resource_id"
+	LabelExternalID    = "external_id"
```

Comment:
It's not my API, but I'd much prefer a name that makes it clear that it's the external ID of the host. Otherwise it's not clear which object the ID belongs to, and the default that I'd expect if I didn't know better is the container.

Could be any of host_external_id, host_id, node_external_id, node_id, etc.